tests/test_regression.py

```diff
@@ -109,6 +109,61 @@ def test_pdf_to_bioc(data_path: Path, input_file: str, config: dict[str, Any]) -
     assert new_tables == expected_tables
 
 
+@pytest.mark.parametrize(
+    "input_file, config, has_tables",
+    [
+        ("Supplementary/Word/mmc1.doc", DefaultConfig.PMC.load_config(), False),
+    ],
+)
+def test_word_to_bioc(
+    data_path: Path, input_file: str, config: dict[str, Any], has_tables: bool
+) -> None:
+    """Test the conversion of a doc file to a BioC format."""
+    from autocorpus.autocorpus import Autocorpus
+
+    doc_path = data_path / input_file
+    expected_output = doc_path.parent / "Expected Output" / doc_path.name
+    with open(
+        str(expected_output).replace(".doc", ".doc_bioc.json"),
+        encoding="utf-8",
+    ) as f:
+        expected_bioc = json.load(f)
+
+    if has_tables:
+        with open(
+            str(expected_output).replace(".doc", ".doc_tables.json"),
+            encoding="utf-8",
+        ) as f:
+            expected_tables = json.load(f)
```

Comment:
Can you provide another data file which does have tables? Otherwise all the table-related code isn't being tested.

You can also amalgamate the `if has_tables` blocks and put them at the end, so it's tidier.